pkg/state/state.go

```diff
@@ -1282,14 +1284,18 @@ func (st *HelmState) GetReleasesWithOverrides() []ReleaseSpec {
 }
 
 func (st *HelmState) SelectReleasesWithOverrides() ([]Release, error) {
-	rs, err := markFilteredReleases(st.GetReleasesWithOverrides(), st.Selectors)
+	values, err := st.Values()
+	if err != nil {
+		return nil, err
+	}
+	rs, err := markExcludedReleases(st.GetReleasesWithOverrides(), st.Selectors, values)
 	if err != nil {
 		return nil, err
 	}
 	return rs, nil
 }
 
-func markFilteredReleases(releases []ReleaseSpec, selectors []string) ([]Release, error) {
+func markExcludedReleases(releases []ReleaseSpec, selectors []string, values map[string]interface{}) ([]Release, error) {
```

Comment:
Ah nice! "exclude" seems nicer here because I was always struggling about if "filter" means "filter in" or "filter out" 👍

Reply:
Haha! Funny you mention that because I thought the same - I even had to [look it up](https://www.google.com/search?q=define%3Afilter) because I thought the usage was wrong, but surely enough it means to get rid of unwanted items!